phpBB/phpbb/db/migration/data/v400/extensions_composer_4.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v400;
+
+use phpbb\db\migration\migration;
+
+class extensions_composer_4 extends migration
+{
+	public function effectively_installed()
+	{
+		return strpos($this->config['exts_composer_repositories'], 'https://www.phpbb.com/customise/db/composer/40') !== false;
```

Comment:
How about https://www.php.net/manual/en/function.str-contains.php ?

Reply:
I was going to do that was I was instead staying consistent with the convention already set here in the related migrations: https://github.com/phpbb/phpbb/blob/7c8b22e33b11d111d915baa3c2bfd3b863b81299/phpBB/phpbb/db/migration/data/v400/extensions_composer_2.php#L22